libcob/move.c

```diff
@@ -979,7 +996,12 @@ cob_move_display_to_edited (cob_field *f1, cob_field *f2)
 				case '7':
 				case '8':
 				case '9':
+#if 1	/* CHECKME: Why should we have a comma in here, necessary as shown in NIST NC,
+	   (TODO: add this to the internal testsuite, must fail if commented out)
+	   but not skip a period? */
```

Comment:
please add this to the TODO list with a note to possibly do it in 3.x first

Reply:
Added to the TODO list.